src/commands/docker.test.ts

```diff
@@ -55,12 +49,11 @@ describe("commands/docker.ts runWorkflowWithProgress()", () => {
     sandbox.restore();
   });
 
-  it("should show a basic error notification if Docker is not available", async () => {
+  it("should exit early if Docker is not available", async () => {
     isDockerAvailableStub.resolves(false);
 
     await runWorkflowWithProgress();
 
-    assert.ok(showErrorMessageStub.calledOnce);
```

Comment:
Now tested in https://github.com/confluentinc/vscode/blob/3ac569b4ed98f3be5f9855c2f5255095d74d965c/src/docker/configs.test.ts#L86-L123